src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -541,27 +598,19 @@ def run_single_testcase(self, testcase_path):
 
     return result
 
-  def set_environment_variables(self):
-    """Sets environment variables needed by afl."""
-    # Tell afl_driver to duplicate stderr to STDERR_FILENAME.
-    # Environment variable names and values that must be set before running afl.
-    environment.set_value(constants.FORKSRV_INIT_TMOUT_ENV_VAR,
-                          constants.FORKSERVER_TIMEOUT)
-    environment.set_value(constants.FAST_CAL_ENV_VAR, 1)
-    environment.set_value(constants.IGNORE_UNKNOWN_ENVS_ENV_VAR, 1)
-    environment.set_value(constants.SKIP_CRASHES_ENV_VAR, 1)
-    environment.set_value(constants.SKIP_CPUFREQ_ENV_VAR, 1)
-    environment.set_value(constants.BENCH_UNTIL_CRASH_ENV_VAR, 1)
-    environment.set_value(constants.STDERR_FILENAME_ENV_VAR,
-                          self.stderr_file_path)
-
   def afl_setup(self):
     """Make sure we can run afl. Delete any files that afl_driver needs to
     create and set any environmnet variables it needs.
     """
     self.set_environment_variables()
     remove_path(self.stderr_file_path)
 
+  def set_afl_fuzz_executable(self):
+    self._executable_path = self.afl_fuzz_path
+
+  def set_afl_showmap_executable(self):
+    self._executable_path = self.afl_showmap_path
+
```

Comment:
These functions are duplicated in the child class and only used there. I think they can be deleted.